vb-fe/src/components/select/SelectOption.tsx

```diff
@@ -0,0 +1,47 @@
+/* Copyright (C) 2021 Vibey - All Rights Reserved */
+
+import React, { FunctionComponent } from "react";
+import { components } from "react-select";
+
+import { Badge, Box } from "@chakra-ui/react";
+
+import "components/select/Select.scss";
+
+/**
+ * Custom option for tags to display the label and score
+ *
+ * Need to figure out the correct type for the prop to get rid of any
+ *
+ * @param props options props passed by react-select f
+ */
+export const SelectOption: FunctionComponent<any> = (props) => {
+  const { data } = props;
+  return (
+    <components.Option {...props}>
+      <Box>
+        <span className="text-grey-1">#</span> <span>{data.label}</span>
+        <Badge colorScheme="primary" mx={2}>
+          {data.score}
+        </Badge>
+      </Box>
+    </components.Option>
+  );
+};
+
+/**
+ * Custom not found option for tags to display the label and score.
+ * We don"t have a "tag not found" result as the not found
+ * tags can be created. So the only time its going to be not found
+ * is when the api is fetching the tags based on the input substring
+ *
+ * Need to figure out the correct type for the prop to get rid of any
+ *
+ * @param props options props passed by react-select
+ */
+export const NoSelectOption: FunctionComponent<any> = (data) => {
+  return (
+    <components.NoOptionsMessage {...data}>
```

Comment:
Is `data` in this case intended to be HTML data like className, etc?  That's how I've typically used a spread in these cases, but I'm not too sure what this `components.NoOptionsMessage` component takes in as props

Reply:
yeah even i'm not too sure, but you have to derive it from react-select and the docs are so trashy i couldn't figure out